docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -428,6 +485,31 @@ The module override index translation algorithm is as follows (**ILAR** = *the n
 
 **TODO**: document attribute presence encoding
 
+**Note**: We already know this table uses assembly-relative token encoding so it has similar
+characteristics like `READYTORUN_SECTION_AVAILABLE_TYPES` or `READYTORUN_SECTION_METHOD_ENTRYPOINTS`.
+No matter what component assembly-relative encoding we end up choosing for these tables, we
+should use the same encoding for ATTRIBUTEPRESENCE.
+
+## READYTORUN_SECTION_ASSEMBLIES (v3.2+)
```

Comment:
A table above states a different version:

> Image (added in V4.1)

Reply:
Fixed in 4th iteration.